### ISSUE TITLE
elvish slow down syntastic

### ISSUE THREAD

**Issue by huiyiqun** [elves/elvish#269]
with syntastic
```
~> before=(date +%s.%N); vim +q; after=(date +%s.%N); - $after $before
▶ 0.22937965393066406
```

without syntastic:
```
~> before=(date +%s.%N); vim +q; after=(date +%s.%N); - $after $before
▶ 1.2602102756500244
```

There is one more second.

This phenomenon does not happen in zsh.

**Comment by xiaq** [comment]
Likely has the same cause with #266.

**Comment by xiaq** [comment]
Has this been fixed by 93726224bfd60f74d7acd7e1dc0495d5a4a86fad?

**Comment by huiyiqun** [comment]
It seems fixed.

However I suggest every elvish user put following in your vimrc:

```
set shell=/usr/bin/sh
```
